acceptancetests/assess_upgrade_series.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+"""Assess upgrading using the 'upgrade-series' commands."""
+
+from __future__ import print_function
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+
+from deploy_stack import (
+    BootstrapManager,
+)
+from jujucharm import (
+    local_charm_path,
+)
+from utility import (
+    add_basic_testing_arguments,
+    configure_logging,
+    JujuAssertionError,
+)
+
+__metaclass__ = type
+
+log = logging.getLogger("assess_upgrade_series")
+charm_bundle = 'upgrade-series.yaml'
+
+
+def assess_juju_upgrade_series(client, args):
+    target_machine = '0'
+    upgrade_series_prepare(client, target_machine, args.to_series, True)
```

Comment:
can we make the True arg agree=True so that it's clear it's the kwarg and what it's doing?

Reply:
¡Si se puede!